src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+
+function getAverageAge(humans) {
+  const agesMap = humans.map(person => person.died - person.born);
+  const averageAges = agesMap.reduce(
+    (prev, next) => prev + next,
+    0) / agesMap.length;
+
+  return averageAges;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const toFilterMan = (person) => {
```

Comment:
What is `toFilterMan` mean? Is it filter function for filter method? Then why not name it like `menFilterFunction`?